package.json

```diff
@@ -5,6 +5,10 @@
   "main": "index.js",
   "typings": "index.d.ts",
   "type": "module",
+  "files": [
+    "index.js",
+    "index.d.ts"
+  ],
```

Comment:
Can you remove this? It makes it harder to maintain and there is no significant difference in size.

Reply:
Ok. Addressed in commit 79252ccf6f5d4cea00c810b2cad2534d16f619d8.